content/doc/pipeline/index.adoc

```diff
@@ -1,981 +1,437 @@
+    * JUnitResultArchiver (junit)
+    * JavadocArchiver (javadoc)
+    * Mailer (mailer)
+    * CopyArtifact (copyartifact)
+    * Analysis publishers
+    * HtmlPublisher (htmlpublisher)
+    * HipChatNotifier (hipchat)
+    * LogParserPublisher (log-parser)
+    * SeleniumHtmlReportPublisher (seleniumhtmlreport)
+    * ScoveragePublisher (scoverage)
+    * AnsiblePlaybookBuilder (ansible)
+    * GitHubCommitNotifier, GitHubSetCommitStatusBuilder (github)
+    * XUnitPublisher and XUnitBuilder (xunit)
+
+Build Wrappers::
+    * ConfigFileBuildWrapper (config-file-provider)
+    * Xvnc (xvnc)
+    * BuildUser (build-user-vars)
+    * TimestamperBuildWrapper (timestamper)
+    * MaskPasswordsBuildWrapper (mask-passwords)
+    * XvfbBuildWrapper (xvfb)
+    * GCloudBuildWrapper (gcloud-sdk)
+    * NpmPackagesBuildWrapper (nodejs)
+    * AnsiColorBuildWrapper (ansicolor)
+
+Triggers::
+    * gerrit-trigger
+    * github
+    * deployment-notification
+    * gitlab-plugin
+    * bitbucket
+
+Clouds::
+    * mock-slave (for prototyping)
+    * docker
+    * Nectar-vmware
+    * operations-center-cloud
+    * ec2
+
+Miscellaneous::
+    * rebuild
+    * parameterized-trigger
+    * build-token-root
+    * job-dsl
+    * zentimestamp
+    * claim
+    * listSubversionTagsParameterValue
+    * authorize-project
+    * customize-build-now
+    * embeddable-build-status
+    * groovy-postbuild
+    * buildtriggerbadge
+    * build-monitor-plugin
+    * radiatorview
+
+Custom Steps::
+    * docker-workflow
+    * credentials-binding
+    * ssh-agen
+    * parallel-test-executor
+    * mailer
+    * email-ext
+
+
+
+*Note:* Several plugins available in the Jenkins ecosystem but not actually related to the Pipeline feature set described in this guide also use the terms "pipeline" or "DSL" in their names. For example:
+
+* Build Pipeline plugin - provides a way to execute Jenkins jobs sequentially
+* Build Flow Plugin - introduces a job type that lets you define an orchestration process as a script. 
+
+=== More Information
+As with any Jenkins plugin, you have the option of installing the Pipeline plugin from the Plugins web page at https://wiki.jenkins-ci.org/display/JENKINS/Plugins, but using the Plugin Manager interface is preferred because you do not then have to make allowances for plugin dependencies or compatibility issues.
+To investigate Pipeline without installing Jenkins separately or accessing your production system, you can run a link:https://github.com/jenkinsci/workflow-plugin/blob/master/demo/README.md[Docker demo] of Pipeline functionality.
+
+== Approaches to Defining Pipeline Script
+You can create pipelines in either of the following ways:
+
+* Through script entered in the configuration page of the user interface for your Jenkins instance.
+* Through script that you create with a Groovy editor outside Jenkins but import into a designated Jenkins repository by selecting the *Pipeline Script from SCM* option during initial setup.
```

Comment:
I don't think this statement fully explains the beauty of the `Jenkinsfile` approach for defining Pipelines. But also "a designated Jenkins repository" I do not think conveys the notion of checking the `Jenkinsfile` in alongside your project's source code.

How about something like:

> Through creation of a `Jenkinsfile` with a text editor, checking it into the project's source control repository and then selecting the "Pipeline Script from SCM" option when configuring the Pipeline in Jenkins.